src/test/scala/fr/polytechnique/cmap/cnam/etl/extractors/patients/PatientsSuite.scala

```diff
@@ -94,7 +94,7 @@ class PatientsSuite extends SharedContext {
       ssr = Some(ssrDf))
 
     // When
-    val result = new Patients(config).extract(sources).toDF.sort("patientID")
+    val result = new Patients(config).extract(sources).toDF
```

Comment:
It can a Dataset[Patient] here instead of a Dataframe.

Reply:
done